engine/src/tangl/service38/hooks.py

```
@@ -0,0 +1,238 @@
+            return transformed
+        if isinstance(value, BaseModel):
+            data = value.model_dump(mode="python")
+            transformed = self._transform_text_fields(data, text_transform)
+            if isinstance(transformed, dict):
+                return value.model_copy(update=transformed)
+            return value
+        if hasattr(value, "text") and isinstance(getattr(value, "text", None), str):
+            try:
+                value.text = text_transform(value.text)
+            except Exception:
+                pass
```

Ruff (0.15.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)